### DISCUSSION TITLE
Prometheus or Prometheus Operator helm chart?

### DISCUSSION THREAD

**Discussion by fr2edms2ekr** [prometheus-operator/prometheus-operator#4131] [Q&A]
Hello all, 

I'm a junior Devops engineer, and although this might come off as a stupid question, I wanted to know which first choice to try deploying, Prometheus or Prometheus Operator? for context I was asked to monitor a VPN behavior installed on a box on the same network as our Kubernetes cluster, and I didn't know if I should just deploy simple Prometheus helm chart or Prometheus Operator helm chart .

Thank you.

**Comment by paulfantom** [comment]
AFAIK there is no prometheus-operator helm chart. 

There is [prometheus helm chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus) and [kube-prometheus-stack helm chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack). The former one installs only prometheus, latter one installs a full monitoring stack for kubernetes including prometheus-operator.

Note: prometheus-operator maintainers are not maintaining any helm charts.

To understand benefits of using prometheus-operator, I recommend going through [getting started document](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md)
